docs/recipes/logging_seq.md

```diff
@@ -0,0 +1,52 @@
+# Logging with Seq
+
+Seq is a popular log-aggregation system that gives you a powerful log search and dashboard engine with views across all of your services.
+
+Tye can push logs to Seq easily without the need for any SDKs or code changes in your services.
+
+## Getting started: running locally with Seq
+
+> :bulb: If you want an existing sample to run, the [sample here](https://github.com/dotnet/tye/tree/master/samples/frontend-backend) will do. This recipe will show examples of UI and data based on that sample. You own application will work fine, but the data and examples will look different.
+
+The first step is to add the `seq` extension to your `tye.yaml`. Add the `extensions` node and its children from the example below.
+
+```yaml
+name: frontend-backend
+
+extensions:
+- name: seq
+  logPath: ./.logs
+
+services:
+- name: backend
+  project: backend/backend.csproj
+- name: frontend
+  project: frontend/frontend.csproj
+```
+
+The `logPath` property here configures the path where Seq will store its data. 
+
+> :bulb: Tye can successfully launch Seq without `logPath`, but ... It's *highly* recommended that you specify a path to store the logs and configuration (add to `.gitignore` if it's part of your repository). Kibana has some mandatory setup the first time you use it, and without persisting the data, you will have to go through it each time.
```

Comment:
Is this actually true for Seq? it looks like this is a copy-paste from the Elastic stack docs

Reply:
Yea there's some copy pasta here.